Dockerfile

```diff
@@ -35,15 +35,15 @@ COPY . .
 RUN bundle exec bootsnap precompile app/ lib/
 
 # Precompiling assets for production without requiring secret RAILS_MASTER_KEY
-RUN SECRET_KEY_BASE=1 ./bin/rails assets:precompile
```

Comment:
this has been working fine, sure about changing it?

Reply:
SECRET_KEY_BASE_DUMMY works now since we are loading rails 7.1 options